src/components/Coupon.tsx

```
@@ -100,13 +90,71 @@ export default function Coupon() {
+        setCoupons(data);
+        console.log(data);
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    getCoupon();
+  }, []);
+
+  useEffect(() => {
+    const selectedRows = table.getSelectedRowModel().rows.map((row) => row.original.issuedCouponId);
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: rowSelection

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)